examples/launch_height_sagemaker.py

```diff
@@ -56,14 +56,10 @@
 
     trial_backend = SageMakerBackend(
         # we tune a PyTorch Framework from Sagemaker
-        sm_estimator=PyTorch(
+        sm_estimator=basic_cpu_instance_sagemaker_estimator(
```

Comment:
You may have to argue with David. He wanted all these examples to be "elementary".